apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -57,13 +50,33 @@ router.get(
     }
 
     const instances = await queryRows(
-      sql.select_lti13_instances,
+      sql.select_combined_lti13_instances,
       {
         course_instance_id: res.locals.course_instance.id,
       },
       Lti13CombinedInstanceSchema,
     );
 
+    if (instances.length === 0) {
+      // See if we have configurations per institution
+      const lti_instances = await queryRows(
```

Comment:
Let's try and preserve naming conventions:
```suggestion
      const lti13_instances = await queryRows(
```